.circleci/config.yml

```diff
@@ -1523,6 +1544,8 @@ workflows:
             - op-batcher-docker-build
             - op-proposer-docker-build
             - op-challenger-docker-build
+      - check-generated-mocks-op-node
```

Comment:
looks like the spacing is off here

Reply:
I think the diff is just cut off making it look that way, see below. I.e. line `1546` is part of the `requires` section of another job in the `main` workflow.
https://github.com/ethereum-optimism/optimism/blob/be65577495e6e19f111beaa9eb3295c585821989/.circleci/config.yml#L1538-L1548

I added the new jobs to the end of the list, but if it's preferable to have them grouped with other relevant jobs I'm happy to move them around!